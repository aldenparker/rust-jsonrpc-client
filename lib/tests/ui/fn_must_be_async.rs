#[jsonrpc_client::api]
pub trait Math {
    fn subtract(&self, subtrahend: i64, minuend: i64) -> i64;
}

fn main() {}
