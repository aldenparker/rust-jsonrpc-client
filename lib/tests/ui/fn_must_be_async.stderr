error: trait methods have to be async
 --> $DIR/fn_must_be_async.rs:3:5
  |
3 |     fn subtract(&self, subtrahend: i64, minuend: i64) -> i64;
  |     ^^
